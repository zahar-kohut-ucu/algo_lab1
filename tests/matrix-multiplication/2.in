2 2
55 70 
58 73 
2 2
43 32 
65 49 
