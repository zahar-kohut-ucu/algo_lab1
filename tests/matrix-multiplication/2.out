6915 5190 
7239 5433 
